components/Actions/Action/Action.module.css

```diff
@@ -1,41 +1,60 @@
 $aspect-ratio: 0.75;
 
 .tile {
-	background-color: $dark-grey;
-	border-radius: 20px;
-	padding: 16px 20px 36px;
-	width: 100%;
-	min-height: 132px;
-	margin-bottom: 8px;
-
-	@media $(medium) {
-		min-height: 184px;
-		padding: 20px 24px 56px;
-		margin-bottom: 0;
-	}
-
-	&.isContact {
-		background-color: transparent;
-		border: 1px solid white;
-		min-height: 56px;
-		@media $(medium) {
-			min-height: 184px;
-		}
-	}
+  position: relative;
+  background-color: $dark-grey;
+  border-radius: 20px;
+  padding: 16px 20px 36px;
+  width: 100%;
+  min-height: 132px;
+  margin-bottom: 8px;
+  border: 1px solid transparent;
+
+  @media $(medium) {
+    min-height: 184px;
+    padding: 20px 24px 56px;
+    margin-bottom: 0;
+  }
+
+  &.isContact {
+    background-color: transparent;
+    border: 1px solid white;
+    min-height: 56px;
+    @media $(medium) {
+      min-height: 184px;
+    }
+  }
+
+  &.hasContext {
+    cursor: pointer;
+
+    &:hover {
+      border: 1px solid white;
+    }
+  }
 }
 
 .title {
-	@mixin subtitle;
-	padding: 0;
-	margin: 0 0 32px;
+  @mixin subtitle;
```

Comment:
These whitespace changes do add weight to the theory your editor maybe autoformatted the `yaml` files